src/contracts/BConst.sol

```diff
@@ -24,4 +24,12 @@ contract BConst {
 
   uint256 public constant MAX_IN_RATIO = BONE / 2;
   uint256 public constant MAX_OUT_RATIO = (BONE / 3) + 1 wei;
+
+  // sending this somewhere far away so it doesn't clash with possible future
+  // _transient_ variables. value is
+  // uint256(keccak256('BPool.transientStorageLock')) - 1;
+  uint256 internal constant _MUTEX_TRANSIENT_STORAGE_SLOT =
```

Comment:
I'm making them internal since storage slot can't be accessed from outside anyway

Reply:
move the constant to BConst? similar to how `COMMITMENT_SLOT` is declared 🙏

Reply:
wdym? it's already on `BConst`